oslo/torch/_C/csrc/fused_triang_softmax.cu

```diff
@@ -0,0 +1,99 @@
+/* coding=utf-8
```

Comment:
How about rename the file name as `fused_upper_triang_softmax.cu`?
header file will be `~.h` with same name.

Reply:
fused_softmax -> scaled_masked_softmax
...

In the past, when I implemented similar code, I wanted to use abbreviated names, but now it seems like it would be better to use name that everyone is familiar with.

Reply:
Okay!